draft-ietf-quic-multipath.md

```diff
@@ -979,7 +979,10 @@ faster if the scheduling strategy is stable, but besides that
 implementations can choose between multiple strategies such as sending
 PATH_ACK frames on the path they acknowledge packets, or sending
 PATH_ACK frames on the shortest path, which results in shorter control loops
-and thus better performance.
+and thus better performance. As packets that only carries PATH_ACK frames
+are not congestion controlled, sending only PATH_ACK frames on a path
+should carefully consider the load induced by these packets, especially
+if the capacity is unknown on that path.
```

Comment:
Typo. Should be " As packets that only carry", not "carries" (subject is plural).